Cargo.toml

```diff
@@ -394,3 +394,8 @@ opt-level = 3
 opt-level = 3
 [profile.release.package.zstd-sys]
 opt-level = 3
+
+# Until https://github.com/ebkalderon/tower-lsp/pull/401 gets landed, we're
+# patching it in.
+[patch.crates-io]
+tower-lsp = { git = "https://github.com/denoland/tower-lsp.git", rev = "910e0f8" }
```

Comment:
We can't publish to crates.io with this. Are we just not going to publish with this PR for now?

Reply:
Why can't we publish to crates.io with this one?

Reply:
IIRC it will error because all the dependencies aren't on crates.io (and I would really hope it has this behaviour)

Reply:
Got it, then we should publish our fork to crates.io

Reply:
Published as https://crates.io/crates/deno_tower_lsp